src/petals/bloom/ops.py

```diff
@@ -116,7 +116,7 @@ def dropout_add(x, residual, prob, training):
     Args:
         x (`torch.tensor`, *required*):
             input tensor
-        residual (`torch.tensor`, *rquired*):
+        residual (`torch.tensor`, *required*):
```

Comment:
this might be on HF

Reply:
Yes, but it got fixed in https://github.com/huggingface/transformers/commit/b69a62d579ded31d490e0c9a01bf9ecb81cb9b65 (by the way, we might want to pull upstream changes at some point)